### DISCUSSION TITLE
Suspense - delay transition

### DISCUSSION THREAD

**Discussion by tomaszferens** [solidjs/solid#396] [Q&A] [answered]
By looking at the demo from examples (https://codesandbox.io/s/solid-suspense-tabs-vkgpj) I noticed that transition to the tab happens after the data has been fetched. Is it possible to transition if the data takes more than e.g. 200ms to load?

Edit: Oh, okay, that might be related: https://github.com/facebook/react/pull/19703

**Comment by ryansolid** [comment, marked as answer]
Exactly. I tend to agree with Andrew and was already thinking this before they made this change. It simplifies the details a lot. The real idea here is we want to avoid the skeleton state if possible. I realize this pushes timeout considerations back to you which is definitely more complicated, but also infinitely flexible once you go that way.
